temporal/t.copy/t.copy.html

```diff
@@ -0,0 +1,46 @@
+<h2>DESCRIPTION</h2>
+
+The purpose of <b>t.copy</b> is to create a copy of a space
+time raster dataset. The new space time dataset will be located in the
+current mapset, whereas the old space time dataset can be located in a
+different mapset as long as this mapset is in the search path.
+
+<p>
+If the <em>-c</em> flag is given, the maps of the old space time dataset
+will also be copied to the current mapset, otherwise the original maps
+will be simply registered in the temporal database. The new copies will
+have the same name like the old maps.
+
+<h2>NOTES</h2>
+A fully qualified name for the input space-time dataset is only required
+if space-time datasets with the same name exist in different mapsets.
+
+<h2>EXAMPLE</h2>
+
+In the North Carolina sample dataset with the separately available
+mapset <em>modis_lst</em> included, copy the space-time raster dataset
+<em>LST_Day_monthly@modis_lst</em> to the current mapset <em>user1</em>:
+
+<div class="code"><pre>
+g.mapsets mapset=modis_lst operation=add
+t.copy input=LST_Day_monthly type=strds output=LST_Day_monthly
```

Comment:
Here, no `input=LST_Day_monthly@modis_lst` is required, because `modis_lst` is added to the mapset search path? (Just out of curiosity, I am not too familiar with `g.mapsets operation=add`)